Source/HelixToolkit.Maths/Vector3.cs

```diff
@@ -148,6 +144,19 @@ public static bool IsZero(this Vector3 v)
             return v.X == 0 && v.Y == 0 && v.Z == 0;
         }
 
+        /// <summary>
+        /// Determines whether the specified vector is undefined (NaN,NaN,NaN).
+        /// </summary>
+        /// <param name="v">The vector.</param>
+        /// <returns>
+        /// <c>true</c> if the specified vector is undefined; otherwise, <c>false</c>.
+        /// </returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool IsUndefined(this Vector3 v)
```

Comment:
Comment suggests that it returns true if all (X, Y, Z) are NaN, but it returns true if any of them is NaN. I'm also wondering what the difference is with method AnyUndefined() on line 879. What it maybe the intention to use && for this method and || for the other?

Reply:
Comment is wrong